### ISSUE TITLE
Can not login using ADAL plugin on windows for some users

### ISSUE THREAD

**Issue by GauravMahajanMicrosoft** [AzureAD/azure-activedirectory-library-for-cordova#92]
Some users particularly on tenant "systemsnationalanalysts.onmicrosoft.com" (where systemsnationalanalysts is the name of the O365 tenant) are prompted for my O365 credentials but the login fails.

Firewall logs show that the computer is trying to reach https://systemsnationalanalysts.onmicrosoft.com(where systemsnationalanalysts is the name of the O365 tenant). That host does not resolve.

Then it tries three times to reach https://autodiscover.systemsnationalanalysts.onmicrosoft.com but that is met with a "Connection refused" error.

The user has a E3 O365 license and are able to login into online solutions which do not require the plugin and uses other libraries for login into AAD.

This is also true for users under "contoso.onmicrosoft.com". 
### Repro Environment Details
1. Installed plugin version: Latest ADAL plugin version: v0.7.1 ( 30de2ba )
2. Target Platform: `Windows 10`
3. Test device and OS version: Windows 10 Desktop x64 
4. SSO is enabled

**Comment by sgrebnov** [comment]
I've tried to reproduce this issue using newly created  Azure AD account and steps similar to what is described in [Connect to Office 365 using ADAL Cordova Plugin](http://taco.visualstudio.com/en-us/docs/access-office365/?rnd=1) (except SSO part - special url to enable SSO) and it works fine. I've also performed step-by-step debugging to ensure native methods are executed w/ correct parameters. I see that there were [several improvements added to native ADAL library](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/releases) from our last update so I propose to bump native library to the latest available to see if it can help and also ask `askADAL` to help w/ this issue as they have more expertise and may have some ideas why this could happen.
